core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,796 @@
-<div class="oppia-about-page-container">
-  <div class="oppia-static-header">
-    <h1 [innerHTML]="'I18N_ABOUT_PAGE_HEADING' | translate"></h1>
+<div class="about-page">
+  <div class="oppia-about-page-section-one">
+    <div class="oppia-about-section-one-image oppia-about-page-section-one-right">
+      <picture class="oppia-about-study-image">
+        <source type="image/webp" [srcset]="getStaticImageUrl('/about/study.webp')">
+        <source type="image/png" [srcset]="getStaticImageUrl('/about/study.png')">
+        <img [src]="getStaticImageUrl('/about/study.png')" alt="about study image">
+      </picture>
+    </div>
+    <div class="oppia-about-page-section-one-left">
+      <h2 class="oppia-about-title oppia-about-page-title" [innerHTML]="'I18N_ABOUT_PAGE_TITLE_SECTION_ONE' | translate"></h2>
+      <p class="oppia-about-content oppia-about-page-content" [innerHTML]="'I18N_ABOUT_PAGE_SECTION_ONE_CONTENT' | translate"></p>
+      <div class="oppia-about-button-container">
+        <button type="submit" (click)="onClickVisitClassroomButton()"
+                class="btn oppia-about-button oppia-about-button-browse oppia-about-browse-button-position oppia-transition-200"
```

Comment:
Are we expecting the button style to be the same/consistent throughout the page?

Reply:
yes we expect the button style to be same throughout the page. just with an exception of button in last section with greater width.

Reply:
@ReshuKumari Is this resolved?

Reply:
yes the button style is same everywhere except the button in seventh section where width is greater.

Reply:
> oppia-about-button-browse

Then why do we have button specific class name? Can't we have a general name for the button class?

> except the button in seventh section where width is greater.

Why can't we set the min-width and have one class for all the buttons, will that be possible?